src/main/java/clzzz/helper/commons/util/DateTimeUtil.java

```diff
@@ -21,4 +22,71 @@
     public static final String BACK_UP_PATTERN = "yyyyMMdd_HH_mm_ss";
     public static final DateTimeFormatter BACK_UP_FORMAT =
             DateTimeFormatter.ofPattern(BACK_UP_PATTERN);
+
+    /**
+     * Check if the given date is a valid date.
+     */
+    public static boolean isValidDateString(String date) {
+        String[] parsedDate = date.split("/");
+        int day = Integer.parseInt(parsedDate[0]);
+        int month = Integer.parseInt(parsedDate[1]);
+        int year = Integer.parseInt(parsedDate[2]);
+        return isValidDate(day, month, year);
+    }
+
+    /**
+     * Check if the given time is a valid time.
+     */
+    public static boolean isValidTimeString(String time) {
+        int timeInt = Integer.parseInt(time);
+        if (timeInt < 0 || timeInt >= 2400) {
```

Comment:
I just checked, STRICT works for time also.
And it rejects 24 cos that'd be 0.

Reply:
Oops you're rejecting 24 also, sorry it's late and I'm tired 😓

Reply:
same reason as above

Reply:
Should’ve expected that a math major would know how to handle 24, unlike me until I ran the code.
But similar to the above we can also relax the format

Reply:
😬